test/integration/image-optimizer/test/sharp.test.ts

```diff
@@ -5,5 +5,5 @@ const appDir = join(__dirname, '../app')
 const imagesDir = join(appDir, '.next', 'cache', 'images')
 
 describe('with latest sharp', () => {
-  setupTests({ appDir, imagesDir })
+  setupTests({ appDir, imagesDir, isrFlushToDisk: false })
```

Comment:
We need to keep one of these with `isrFlushToDisk: true` or else we won't be running all the tests

```suggestion
  setupTests({ appDir, imagesDir, isrFlushToDisk: true })
```